software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcessWinRmDriver.java

```diff
@@ -49,15 +50,28 @@ public void start() {
 
     @Override
     public void install() {
+        String hostname = null;
```

Comment:
At some point in the future, this should probably be refactored to get the name of the machine in `WinRmMachineLocation` and set it as the hostname sensor. Please add a TODO note here suggesting the same